ext/bcmath/libbcmath/src/long2num.c

```diff
@@ -0,0 +1,59 @@
+/*
+   +----------------------------------------------------------------------+
+   | Copyright (c) The PHP Group                                          |
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 3.01 of the PHP license,      |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available through the world-wide-web at the following url:           |
+   | https://www.php.net/license/3_01.txt                                 |
+   | If you did not receive a copy of the PHP license and are unable to   |
+   | obtain it through the world-wide-web, please send a note to          |
+   | license@php.net so we can mail you a copy immediately.               |
+   +----------------------------------------------------------------------+
+   | Authors: Saki Takamachi <saki@php.net>                               |
+   +----------------------------------------------------------------------+
+*/
+
+#include "bcmath.h"
+#include <stdbool.h>
+#include "convert.h"
+
+#define BC_LONG_MAX_DIGITS sizeof(LONG_MIN_DIGITS) - 1
+
+bc_num bc_long2num(zend_long lval)
+{
+	bc_num num;
+
+	if (UNEXPECTED(lval == 0)) {
+		num = bc_copy_num(BCG(_zero_));
+		return num;
+	}
+
+	bool negative = lval < 0;
+	if (UNEXPECTED(lval <= LONG_MIN)) {
```

Comment:
Comparison `==` doesn't work properly on Windows. `<=` works fine, I don't know why...

Reply:
OK, I understand.
I should use `ZEND_LONG_MIN`